Datastore/src/Query/AggregationQueryResult.php

```diff
@@ -162,12 +174,46 @@ public function getTransaction()
     }
 
     /**
-     * Gets the timsstamp when Aggregation was executed.
+     * Gets the timestamp when Aggregation was executed.
      *
      * @return Timestamp
      */
     public function getReadTime()
     {
         return $this->readTime;
     }
+
+    /**
+     * Gets the ExplainMetrics object if the explainOptions parameter was set for
+     * the query
+     *
+     * @return null|ExplainMetrics
+     */
+    public function getExplainMetrics(): null|ExplainMetrics
+    {
+        return $this->explainMetrics;
+    }
+
+    private function fixDurationFormat(array $metrics): array
+    {
+        // The current protobuf library does not support the current json representation
+        // of the well-known type Duration.
+        // Hence we have to convert the object format into a string format for the merging from json to work.
+        // If the protobuf library gets updated, this should be removed.
+        if (!isset($metrics['executionStats']) && !isset($metrics['executionStats']['executionDuration'])) {
```

Comment:
This should be OR, not AND... if the first is true, then the second is always true. 
```suggestion
        if (!isset($metrics['executionStats']) || !isset($metrics['executionStats']['executionDuration'])) {
```